tests/pipeline/test.nf.test

```diff
@@ -14,7 +14,37 @@ nextflow_pipeline {
         }
 
         then {
-            assert workflow.success
+            assertAll(
+                { assert workflow.success },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
+
+                { assert snapshot(path("$outputDir/db_tables/alleles.tsv"),
+                                path("$outputDir/db_tables/conditions_alleles.tsv"),
+                                path("$outputDir/db_tables/conditions.tsv"),
+                                path("$outputDir/db_tables/entities_proteins.tsv"),
+                                path("$outputDir/db_tables/entities.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.no_weights.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.nucl.tsv"),
+                                path("$outputDir/db_tables/microbiomes_entities.tsv"),
+                                path("$outputDir/db_tables/microbiomes.tsv"),
+                                path("$outputDir/db_tables/proteins_peptides.tsv"),
+                                path("$outputDir/db_tables/stats.txt")).match("db_tables") },
+                { assert new File("$outputDir/db_tables/peptides.tsv.gz").exists() },
+                { assert new File("$outputDir/db_tables/predictions.tsv.gz").exists() },
+                { assert new File("$outputDir/db_tables/proteins.tsv.gz").exists() },
```

Comment:
The predicted peptides/proteins are not reproducible? Seems a little worrying, any idea why? Or is it just the sorting?

Reply:
It should be just the sorting as the number of peptides/proteins is not changing (that's in the stats.txt file) but I will check the files again to be sure!

Reply:
if its indeed sorting and if it would be (easily) doable, sort the table

Reply:
it seems to be the same case for some other output files, right?

Reply:
Yes for `alleles.tsv` I could already easily fix it. I'm checking the content of the files again to see if I can solve some md5sum issues!